keyboards/ergodox_ez/keymaps/hacker_dvorak/config.h

```diff
@@ -0,0 +1,76 @@
+/*
+  Set any config.h overrides for your specific keymap here.
+  See config.h options at https://docs.qmk.fm/#/config_options?id=the-configh-file
+*/
+#ifndef CONFIG_HACKER_DVORAK_H
+#define CONFIG_HACKER_DVORAK_H
+
+#include "../../config.h"
+
+// Keyboard Settings
+#undef TAPPING_TERM
+#define TAPPING_TERM 1000
+
+#undef DEBOUNCE
+#define DEBOUNCE 10
+
+#undef ONESHOT_TAP_TOGGLE
+#define ONESHOT_TAP_TOGGLE 1
```

Comment:
This is more of a personal preference, but setting this to one may not be the best idea. 2 works very well.  But if this works well for you, feel free to ignore.